From 99157838d346a1f97d61f1756698dbad7954d085 Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Thu, 8 Nov 2012 18:32:52 -0800
Subject: [PATCH] gpio: xilinx_gpiops: Use dev_err over pr_err

Replacing uses of pr_err through dev_err.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
Acked: Michal Simek <michal.simek@xilinx.com>
---
 drivers/gpio/gpio-xilinxps.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpio/gpio-xilinxps.c b/drivers/gpio/gpio-xilinxps.c
index 5849500..787b816 100644
--- a/drivers/gpio/gpio-xilinxps.c
+++ b/drivers/gpio/gpio-xilinxps.c
@@ -574,13 +574,13 @@ static int __init xgpiops_probe(struct platform_device *pdev)
 	/* Enable GPIO clock */
 	gpio->clk = clk_get_sys("GPIO_APER", NULL);
 	if (IS_ERR(gpio->clk)) {
-		pr_err("Xilinx GPIOPS clock not found.\n");
+		dev_err(&pdev->dev, "Clock not found.\n");
 		ret = PTR_ERR(gpio->clk);
 		goto err_chip_remove;
 	}
 	ret = clk_prepare_enable(gpio->clk);
 	if (ret) {
-		pr_err("Xilinx GPIOPS unable to enable clock.\n");
+		dev_err(&pdev->dev, "Unable to enable clock.\n");
 		goto err_clk_put;
 	}
 #endif
-- 
1.7.0.4

